### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by zkrolikowski-vl**
Two changes:

- added usage example
- extended the testing section with tox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,38 @@ Alternatively, if you want a cleaner `PYTHONPATH` or wish to modify the annotati
...
+```
+import pandas as pd
+
+decimals = pd.DataFrame({'TSLA': 2, 'AMZN': 1})
+prices = pd.DataFrame(data={'date': ['2021-08-13', '2021-08-07', '2021-08-21'],
+                            'TSLA': [720.13, 716.22, 731.22], 'AMZN': [3316.50, 3200.50, 3100.23]})
+rounded_prices = prices.round(decimals=decimals)
+```
+
+mypy won't see any issues with that, but after installing pandas-stubs and running it again
```
Comment (joannasendorek) [inline README.md:52]:
```suggestion
mypy won't see any issues with that, but after installing pandas-stubs and running it again
```

File: README.md [hunk 2]
```diff
@@ -36,6 +36,38 @@ Alternatively, if you want a cleaner `PYTHONPATH` or wish to modify the annotati
...
+
+```
+mypy round.py
+```
+
+we get the following error message
```
Comment (joannasendorek) [inline README.md:58]:
```suggestion
we get the following error message:
```

File: README.md [hunk 3]
```diff
@@ -36,6 +36,38 @@ Alternatively, if you want a cleaner `PYTHONPATH` or wish to modify the annotati
...
+
+```
+round.py:6: error: Argument "decimals" to "round" of "DataFrame" has incompatible type "DataFrame"; expected "Union[int, Dict[Union[int, str], int], Series]"
+```
+
+And after confirming with the [docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.round.html)
+we can fix the code
```
Comment (joannasendorek) [inline README.md:65]:
```suggestion
we can fix the code:
```

File: README.md [hunk 4]
```diff
@@ -46,9 +78,32 @@ The versions follow a pattern `MAJOR.MINOR.PATCH.STUB_VERSION` where the first t
 
 ## Type checkers
 
-As of now Mypy is the only type checker the stubs were tested with. 
+As of now Mypy is the only type checker the stubs were tested with.
```
Comment (joannasendorek) [inline README.md:81]: Sometimes you use small letter for mypy and sometimes capital.

File: README.md [hunk 5]
```diff
@@ -46,9 +78,32 @@ The versions follow a pattern `MAJOR.MINOR.PATCH.STUB_VERSION` where the first t
...
+
+## Development
+
+### Testing using tox
+
+Tox will automatically run all types of tests mentioned further.
+It will create temporary isolated environments for each version of Python and install `pandas-stubs`
```
[the remark this answers is not in the document]
Comment (joannasendorek) [reply README.md:88]: "Each version" -> "Each declared version"?

I mean, it is not each version technically.

File: README.md [hunk 6]
```diff
@@ -46,9 +78,32 @@ The versions follow a pattern `MAJOR.MINOR.PATCH.STUB_VERSION` where the first t
...
+like it would normally be installed when using pip or conda.
+
+Using it is as simple as:
```
Comment (joannasendorek) [inline README.md:91]:
```suggestion
Usage is as simple as:
```

File: README.md [hunk 7]
```diff
@@ -62,12 +117,19 @@ Test the stub files against actual code examples (this will use the stubs from t
...
 
-Test the **installed** stub files against actual code examples (this will use pandas .pyi files from your env):
+Test the **installed** stub files against actual code examples.
+You'll need to install the library beforehand - the .pyi files from your env will be used:
 
 ```
 mypy --config-file mypy_env.ini tests/snippets
 ```
 
+Test the code examples themselves:
```
Comment (joannasendorek) [inline README.md:127]: Why "themselves"?

File: README.md (hunk 2 above)
Comment (zkrolikowski-vl) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (zkrolikowski-vl) [reply README.md]: Fixed. I'll use the capital letter only at the beginning of sentences.

File: README.md (hunk 7 above)
Comment (zkrolikowski-vl) [reply README.md]: Changed

**Review by joannasendorek — APPROVED** [review]

**Pull request merged**